python-client/tests/test_push.py

```diff
@@ -126,18 +128,35 @@ def test_slicing_function_multi(enron_model, enron_data):
             assert isinstance(push.slicing_function, QueryBasedSliceFunction)
 
 
-def test_test_function_multi(enron_model, enron_data):
+def test_test_function_and_output(enron_model, enron_data):
+
+    # Define a list of push types and corresponding functions
+    push_types = [
+        (giskard.push.ContributionPush, create_contribution_push),
+        (giskard.push.PerturbationPush, create_perturbation_push),
+        (giskard.push.OverconfidencePush, create_overconfidence_push),
+        (giskard.push.BorderlinePush, create_borderline_push),
+    ]
+
+    # Initialize dictionaries to store outputs
+    outputs = {push_type: [] for push_type, _ in push_types}
+
+    # Loop through the range and create push outputs
     for i in range(50):
```

Comment:
I'm not a big fan of blindly taking 50 first rows of enron. In this case we're not really sure what we're testing on. If we know which input leads to which push generation we should use it explicitly.
For example you can take row #5 that is known to generate 2 pushes and assert that both of them are generated and contain expected results.